conans/test/functional/toolchains/cmake/test_ninja.py

```diff
@@ -1,144 +1,130 @@
-        """ Ninja build must proceed using default profile and cmake build (Windows Release)
-        """
-        client = TestClient(path_with_spaces=False)
-        client.save({"conanfile.py": self.conanfile,
-                     "main.cpp": self.main_cpp,
-                     "CMakeLists.txt": self.cmake})
-        win_host = textwrap.dedent("""
-            [settings]
-            os=Windows
-            arch=x86_64
-            compiler=Visual Studio
-            compiler.version=15
-            compiler.runtime=MD
-            build_type=Release
-             """)
-        client.save({"win": win_host})
-        client.run("install . -pr=win")
-        # Ninja is single-configuration
-        vcvars = vcvars_command("15", architecture="amd64")
-        client.run_command('{} && cmake . -G "Ninja" -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake '
-                           .format(vcvars))
-        client.run_command("{} && cmake --build .".format(vcvars))
-        client.run_command("App")
-        self.assertIn("main: Release!", client.out)
-        self.assertIn("main _M_X64 defined", client.out)
-        self.assertIn("main _MSC_VER19", client.out)
-        self.assertIn("main _MSVC_LANG2014", client.out)
-
-        check_vs_runtime("App.exe", client, "15", build_type="Release", static=False)
-
-    @pytest.mark.skipif(platform.system() != "Windows", reason="Only windows")
-    def test_locally_build_windows_debug(self):
-        """ Ninja build must proceed using default profile and cmake build (Windows Debug)
-        """
-        client = TestClient(path_with_spaces=False)
-        client.save({"conanfile.py": self.conanfile,
-                     "main.cpp": self.main_cpp,
-                     "CMakeLists.txt": self.cmake})
-        win_host = textwrap.dedent("""
-            [settings]
-            os=Windows
-            arch=x86
-            compiler=Visual Studio
-            compiler.version=15
-            compiler.runtime=MTd
-            build_type=Debug
-             """)
-        client.save({"win": win_host})
-        client.run("install . -pr=win")
-        # Ninja is single-configuration
-        # It is necessary to set architecture=x86 here, otherwise final architecture is wrong
-        vcvars = vcvars_command("15", architecture="x86")
-        client.run("install . -pr=win")
-        client.run_command('{} && cmake . -G "Ninja" -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake '
-                           .format(vcvars))
-        client.run_command("{} && cmake --build .".format(vcvars))
-        client.run_command("App")
-        self.assertIn("main: Debug!", client.out)
-        self.assertIn("main _M_IX86 defined", client.out)
-        self.assertIn("main _MSC_VER19", client.out)
-        self.assertIn("main _MSVC_LANG2014", client.out)
-
-        check_vs_runtime("App.exe", client, "15", build_type="Debug", static=True)
+from conans.test.functional.toolchains.ios._utils import create_library
```

Comment:
Do not import from a private file.
Just use the ``gen_function`` helpers and write a short CMakeLists.txt here, so this is self contained.
We will make something similar to ``gen_function`` and ``gen_makefile_am`` for CMake too, but that better in a separate PR.